images/clio/Dockerfile

```diff
@@ -1,12 +1,11 @@
 FROM ubuntu:20.04 as build
 
-ARG VERSION=0.2.0
+ARG CLIO_VERSION=0.2.0
+ARG BOOST_VERSION=75
```

Comment:
Seems like it would make sense for the boost version to match the package convention, ie. 1_75_0. 
This way, the argument can be easily updated for future versions. What do you think?

Reply:
I see. My proposed correction will not work since there are two formats within the boost tarball.

Reply:
I'd prefer that, but I didn't feel like doing version gymnastics there for no payoff.